packages/align-deps/src/cli.ts

```diff
@@ -180,6 +187,7 @@ async function makeCommand(args: Args): Promise<Command | undefined> {
     write,
     excludePackages: excludePackages?.toString()?.split(","),
     requirements: requirements?.toString()?.split(","),
+    diffMode,
```

Comment:
I think this would be a good place to validate that we have a correct value (or none) for diff mode:
```suggestion
    diffMode: validateDiffMode(diffMode),
```

Reply:
Can you help me a little bit with the validation though?

My initial idea was (when I brought up validation earlier) was:
```ts
function isDiffMode(mode: string): mode is DiffMode {
  return mode === "strict" || mode === "allow-exact-version";
}
```

However this expects a string and we have `string | undefined` here.

Reply:
I would do something like this:

```ts
function validateDiffMode(mode: string | undefined): DiffMode {
  switch (mode) {
    case "allow-exact-version":
    case "strict":
      return mode;
    default:
      if (mode) {
        warn("Unknown diff mode:", mode);
      }
      return "strict";
  }
}
```

Reply:
Thanks, applied it via rebase.